flytekit/common/tasks/sagemaker/distribution.py

```diff
@@ -0,0 +1,99 @@
+import json as _json
```

Comment:
the name of the file is weird?
distributed_training.py?

Reply:
fixed